WCON_format.md

```diff
@@ -79,36 +79,38 @@ Any numeric quantity followed on a per-animal or per-timepoint basis must have i
 
 For compound units, `*` can be used for multiplication, `/` for division, and `^` for exponentiation.  For instance, a unit of acceleration could be written `"mm^2/s"`.  Use `"1"` for a unitless quantity (or `"%"`, as appropriate).  `""` is treated as `"1"`.
 
-Other numerical values may be used in the expressions: for instance, `"0.04*s"` can be used for frame indices in a 25 FPS sample.
+Other numerical values may be used in the expressions: for instance, `"t":"0.04*s"` would allow frame indices in a data set gathered at 25 FPS to be interpreted as times.  Likewise, a program could report coordinates as raw pixel values and set "x" and "y" units accordingly.
 
-`"units"` is required and must be single-valued.  Units must be specified for all quantities used.  If the data set is non-empty, that means there must be units at least for `"t"`, `"x"`, and `"y"` since those are required.  It is permitted to specify units for variables that do not exist.  An empty `"data"` section is also permitted, but WCON readers are allowed to fail on this degenerate case if units for `"t"`, `"x"`, and `"y"` are not supplied.
+`"units"` is required and must be single-valued.  Units must be specified for all quantities used.  If the data set is non-empty, that means there must be units at least for `"t"`, `"x"`, and `"y"` since those are required.  It is permitted to specify units for variables that do not exist.  Although an empty `"data"` section is permitted (meaning there is no `"t"`, `"x"`, or `"y"`), but WCON readers are allowed to fail on this degenerate case if units for `"t"`, `"x"`, and `"y"` are not supplied.
 
 #### Data
 
-The `"data"` entry contains data for one or more animals at one or more timepoints.  It may be arrayed or single-valued.
+The `"data"` entry contains data for one or more animals at one or more timepoints.  It may be arrayed.
 
-A data entry is an object with four required keys.
+A single data entry is an object with four required keys.
 
-1. Each worm is identified by a numerical `id` so that multiple worms can be distinguished in one data set.  The `id` must be single-valued (JSON numbers or strings).
+1. Each worm is identified by an `id` field containing a JSON string.  (Software that generates numeric IDs should convert the number to its corresponding string.)
 
-2. The time of a recording is specified by `t` (a JSON number).  It may be single-valued, for a single timepoint, or arrayed for a series of timepoints.
+2. The time of a recording is specified by `t` (a JSON number).  It may be single-valued, for a single timepoint, or arrayed for a series of timepoints in which case they should be increasing in value.  Every other numeric value must have the same number of entries as `t` has, where the `i`th entry corresponds to the value(s) at the `i`th time-point.
```

Comment:
The issue here is whether `t` is "arrayed", which we specify as admitting a singleton instead of an array with a single element.  In this way, `t` would be a bit weird if it had to be an array.  But it does complicate things to handle the singleton case, since dimensions all have to match.  So I agree--this _must_ be an array.  This makes custom splitting/joining simpler, too: if it's an array, split it.  If not, don't.